src/server/node_recov_db.c

```diff
@@ -173,8 +176,11 @@ node_recov_db(char *nd_name, struct pbsnode *pnode)
 
 	if (rc == 0)
 		rc = db_to_node(pnode, &dbnode);
-	else
-		log_errf(PBSE_INTERNAL, __func__, "Failed to load node %s %s", nd_name, (conn->conn_db_err)? conn->conn_db_err : "");
+	else {
+		pbs_db_get_errmsg(PBS_DB_ERR, &conn_db_err);
+		log_errf(PBSE_INTERNAL, __func__, "Failed to load node %s %s", nd_name, conn_db_err? conn_db_err : "");
```

Comment:
space before conditional operator. Please do the same for other places where you use conditional operator.